backend/src/main/java/com/rota/api/StaffController.java

```diff
@@ -56,4 +64,39 @@
 
     return staffService.getStaffEngagementsBetween(user, start, end);
   }
+
+  /**
+   * Endpoint to create and enter a new staff record into the system.
+   * Staff ID and role is inferred from the token passed in the header's Authorization field.
+   * @param authString Authentication token.
+   * @param createStaffForm Staff details.
+   * @return
+   */
+  @PostMapping("/staff/create")
+  @ApiOperation(value = "Lets an authenticated manager create a new staff user",
+      consumes = "application/json")
+  public ResponseEntity createStaff(
+      @RequestHeader("Authorization")
+      @ApiParam(value = "Authentication token")
+          String authString,
+
+      @Valid
+      @RequestBody
+      @ApiParam("Staff details for the new user")
+          CreateStaffForm createStaffForm
+  ) {
+    final Role role = AuthenticationUtils
+        .getUserRoleFromToken(authString)
+        .orElseThrow(() ->
+          new ResponseStatusException(HttpStatus.UNAUTHORIZED, "Failed to authorize.")
+        );
+
+    if (role != Role.MANAGER) {
```

Comment:
I would replace the content of this if statement with a function e.g. `hasManagerPermissions(role)`.
That way if we ever add more roles that have manager permissions we'll be able to fix that easily.